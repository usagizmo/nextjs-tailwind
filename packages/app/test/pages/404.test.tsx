import React from 'react'
import { render } from '../test-utils'
import Page404 from '../../src/pages/404'

describe('Page404 page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Page404 />)
    expect(asFragment()).toMatchSnapshot()
  })
})
