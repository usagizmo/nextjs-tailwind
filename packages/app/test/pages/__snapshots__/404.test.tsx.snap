// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 page matches snapshot 1`] = `
<DocumentFragment>
  <main
    class="flex items-center justify-center h-full"
  >
    <h1
      class="flex items-center h-16 pr-6 mr-6 border-r border-gray-500 text-2xl font-medium"
    >
      404
    </h1>
    <p
      class="text-sm"
    >
      This page could not be found.
    </p>
  </main>
</DocumentFragment>
`;
